### ISSUE TITLE
Error in twilsock's configuration.d.ts file - cannot find 'loglevel'

### ISSUE THREAD

**Issue by david-bulte** [twilio/twilio-chat-console-webapp.js#3]
This is not a bug in the demo per se but in a dependency, i.e. twilsock. I can't find that repo so I hope by posting this here, somehow it will find its way to the right person.

When building my chat application (an angular/typescript application, not this demo), I get the following error:

> ERROR in node_modules/twilsock/lib/configuration.d.ts(1,23): error TS2688: Cannot find type definition file for 'loglevel'.

The reason is that loglevel no longer exists as a type, it's now a simple string. The version I am using is  "twilio-chat": "^3.0.0",

**Comment by aleksandrsivanovs** [comment]
Hi!

yes, that was the problem in our library called twilsock - but it was fixed, afaik. 
I've encountered the same problem when tried to build React Native example. 
however, it should be fixed by now (at least React Native doesn't complain anymore: https://github.com/twilio/TwilioChatJsReactNative).

I'll try to dig through this week to figure out what's wrong. 
Can you share your code where exactly it is not working, so I could have a reference to work with?

Aleksandrs

**Comment by david-bulte** [comment]
Thanks for your quick response!

I can't share my (in-house) project, but I have created a very simple repo ([](https://github.com/david-bulte/test-chat)) that shows the problem.

It's a standard angular app, with 1 extra dependency, i.e. twilio-chat. In the app.component.ts file, there is a reference to twilio-chat/lib/channel. This triggers the angular build to build the twilio code as well, which leads to aforementioned problem.

So in short do: `npm install` and `ng build`

**Comment by aleksandrsivanovs** [comment]
thanks, I will just re-post the sample repository name: https://github.com/david-bulte/test-chat (so it won't be lost in my email)

**Comment by aleksandrsivanovs** [comment]
Hi!

we have made new release candidate with fix for this issue: `twilio-chat@3.0.1-rc1` (it's published in [twilio-chat](https://www.npmjs.com/package/twilio-chat) package's alpha channel).
I've verified and it seems that it's fixed the issue. 
can you, please, verify from your side. 

if all is good - we will do new release of 3.0.1 version of twilio chat on Monday.

**Comment by david-bulte** [comment]
Yes! Its' working 👍 Thanks!

**Comment by aleksandrsivanovs** [comment]
awesome, we will release today or tomorrow. 
will close this item when we will release

**Comment by aleksandrsivanovs** [comment]
done, 3.0.1 released, closing this item
